.storybook/config.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from "@storybook/react";
+
+// automatically import all files ending in *.stories.js
+configure(require.context("../stories", true, /\.stories\.js$/), module);
```

Comment:
- [ ] change 
```
configure(require.context("../stories", true, /\.stories\.js$/), module);
```
 to
```
const loadStories = () => {
  const req = require.context("../src/Components", true, /.story.js$/);
  req.keys().forEach(filename => req(filename));
};

configure(loadStories, module);
```

This will allow us to put the story in each component folder.

- [ ] Next, because of the above change we can remove the stories folder and put the story file in the Component folder. So this mean stories/DrumPad.stories.js will turn into src/Components/DrumPad/index.story.js. This way we can group the stories with their react code so that things can be more organized.

Reply:
the above code will not load stories are you sure you want to implement it?